README.rst

```diff
@@ -110,11 +112,12 @@ __ `FIWARE Policy Manager GE: Facts`_
 Configuration file
 ------------------
 The configuration used by the fiware-facts component is optionally read from the file
-``conf/fiware-facts.cfg``
+``facts_conf/fiware-facts.cfg``
 
 Default values are found in ``facts/config.py``.
```

Comment:
No, There is only one config file.
Fiware-facts.cfg is an entry point to replace default configuration with the values provided by the user in this file, and it is optional